fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/load/ChunkWriter.java

```diff
@@ -195,7 +195,7 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
                                 if (auditLogger.shouldLog()) {
                                     long endTime = System.currentTimeMillis();
                                     String location = "@source:" + ctx.getSource() + "/" + ctx.getImportPartitionWorkitem();
-                                    auditLogger.logUpdateOnImport(fhirResource, null, new Date(startTime), new Date(endTime), Response.Status.OK, location, "BulkDataOperator");
+                                    auditLogger.logUpdateOnImport(null, fhirResource, new Date(startTime), new Date(endTime), Response.Status.OK, location, "BulkDataOperator");
```

Comment:
Good catch, although its a little funny that this method takes the "oldResource" at all since we don't actually try to read that.  Assuming this is the only place its ever called from, maybe we should change the method signature?

Reply:
I wanted to match as much as possible what's in the RestAuditLogger.